components/slider-carousel/src/ReuseSliderCarousel.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useRef, useEffect } from "react";
+import { useSlider } from "@locoworks/reusejs-toolkit-react-hooks";
+import { twMerge } from "tailwind-merge";
+
+interface ReuseSliderCarouselInterface {
+  slideInterval: number;
+  slides: React.ReactNode[];
+  loop: boolean;
+  wrapperClasses?: string;
+  sliderContainerClasses?: string;
+  enableButtons?: boolean | false;
```

Comment:
Boolean already have true false value. Why is false added her?

Reply:
by default i need false value that's why it is used.
I updated code now it is boolean value.

and assigned default value inside component props.